### PR TITLE
mod_wsgi module doc page

### PR DESCRIPTION
**Pull request by sashakames**
Fixes #

## Proposed Changes

  - Adds a page describing how to add services on a node with mod_wsgi-express manually
  -
  -

### CODE REVIEW

File: docs/source/developers/developers.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    source /usr/local/conda/bin/activate
+    conda create -n flaskdemo
+    conda activate flaskdemo
+
+.. note:: 
+    if your app was developed in Python 2.7 you'll need to create the environment with the following instead:
+
+::
+
+    conda create -n flaskdemo
```
Comment (nathanlcarlson) [inline docs/source/developers/developers.rst]: I think you mean to have "python<3" here

File: docs/source/developers/developers.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Install modules needed to run your app.  Our demo uses flask, but you could use django (and expect additional required packages.)  Note that diffculty has been encountered with mod_wsgi version 4.6.7, so we recommend an earlier version:  ::
+
+::
+
+    pip install flask 'mod_wsgi<4.6'
```
Comment (nathanlcarlson) [inline docs/source/developers/developers.rst]: Need to make the environment is activated

Comment (sashakames) [reply docs/source/developers/developers.rst]: done in previous step

**Comment by nathanlcarlson** [comment]
Is this ready to merge?

**Comment by sashakames** [comment]
I think I responded to your concerns, so, ready for merge.

**Pull request merged**